data_structures/flag_filter.wdl

```diff
@@ -0,0 +1,157 @@
+## # FlagFilter
+##
+## A struct to represent the flags used in the `samtools fastq` command.
```

Comment:
These are also supported by `view`. There may be others as well.

Reply:
That's a good point. Although I'm worried that the order of precedence might not be consistent across subcommands. That's just a hunch. I'll look into it tomorrow and see if we can broaden the scope a bit on what other commands can use this struct.